x-pack/plugins/enterprise_search/public/applications/app_search/components/source_engines/source_engines.test.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import '../../../__mocks__/shallow_useeffect.mock';
+
+import { setMockActions, setMockValues } from '../../../__mocks__';
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { EuiCodeBlock } from '@elastic/eui';
+
+import { Loading } from '../../../shared/loading';
+
+import { SourceEngines } from '.';
+
+const MOCK_ACTIONS = {
+  // FlashMessagesLogic
+  dismissToastMessage: jest.fn(),
+  // SourceEnginesLogiuc
+  fetchSourceEngines: jest.fn(),
+};
+
+const MOCK_VALUES = {
+  dataLoading: false,
+  sourceEngines: [],
+};
+
+describe('SourceEngines', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    setMockActions(MOCK_ACTIONS);
+  });
+
+  describe('non-happy-path states', () => {
+    it('renders a loading component before data has loaded', () => {
+      setMockValues({ ...MOCK_VALUES, dataLoading: true });
+      const wrapper = shallow(<SourceEngines engineBreadcrumb={[]} />);
```

Comment:
We shouldn't need an `engineBreadcrumb` prop anymore with the handy dandy new breadcrumb helper

```suggestion
      const wrapper = shallow(<SourceEngines />);
```
Also, I'm surprised Typescript isn't yelling about this 🤔

Reply:
No it did, I didn't run the linter universally after merging master and fixing the conflict errors, my bad

Reply:
All good!

Reply:
I addressed this in [cc65416](https://github.com/elastic/kibana/pull/96896/commits/cc654166ea55a01918746e7978589df9ca02a644)